.gitignore

```diff
@@ -0,0 +1,19 @@
+.svn/
+.hg/
+.hgtags/
+log
+*.py[cod]
+build
+doc/_build
+dist/
+pylint.egg-info/
+.tox
+*.sw[a-z]
+doc/features.rst
+pyve
+build-stamp
+debian/files
```

Comment:
I have made the ignored files that were explicitly rooted in the hgignore similarly rooted in the gitignore.